sdk/resourcemanager/appservice/armappservice/README.md

```diff
@@ -50,6 +50,12 @@ options = arm.ClientOptions{
 client := armappservice.NewWebAppsClient(<subscription ID>, cred, &options)
 ```
 
+## More sample code
+
+- [Static Site](https://aka.ms/azsdk/go/mgmt/samples?path=web/staticsite)
+- [App Service Plan](https://aka.ms/azsdk/go/mgmt/samples?path=web/web_plans)
+- [App Service](https://aka.ms/azsdk/go/mgmt/samples?path=web/webapp)
```

Comment:
This reminds me that since we renamed web to appservice in the SDK, we should also rename it in the sample repo. @Alancere could you please open a PR to change that?

Reply:
rename to appservice https://github.com/Azure-Samples/azure-sdk-for-go-samples/pull/335